res/ari/ari_model_validators.c

```diff
@@ -1165,6 +1165,15 @@ int ast_ari_validate_channel(struct ast_json *json)
 				res = 0;
 			}
 		} else
+		if (strcmp("tenantid", ast_json_object_iter_key(iter)) == 0) {
```

Comment:
ari_model_validators.c and .h are generated by running `make ari_stubs` after modifying files in /rest-api/api-docs.